documents/Final Report/outlook.tex

```diff
@@ -4,10 +4,6 @@ \chapter{Outlook}
 
-To get our agent to behave more like a human there are some enhancements we could make as well. Our current agent has a very specific strategy which will execute in exactly the same order most of the time. This is something that could be improved, as this makes the agent predictable, where a human would act more randomly. The agent has also very fast decision making, where a human would take its time to respond, the agent responds immediatly to most things. When playing as a human this seems a bit odd, and a delay should be introduced.
-
-\newpage
-\section {Project}
-We did have fun during this project, but there were also some problems which caused a lot of frustration. The biggest issue was probably that we did not have a good understanding about the functionalities of the connector. We expected it to have most basic functionalities, and that we could later implement some more advanced functionalities if needed. But it turned out that we still had to implement most of the functionality. A good improvement for next year would be to start working on the connector in the first week. It would also be very usefull to have a session in which the connector would be explained. Although working with such a big group has its problems, we do think this should still be done in the next years, it was a big learning experience and we learned a lot about communication and working on a software product with a lot of people. An improvement would be to have a weekly meeting with all the groups and the TAs. We did start having these at the end of the project, but these were a bit chaotic sometimes and not every group made good agreements during this meetings. Another improvement would be to have a stable goal version in which tests and the debugger would work consistently, this caused a lot of delays this year. The last few weeks none of the groups were able to use the debugger succesfully in eclipse and this made fixing bugs a lot harder then it has to be.
\ No newline at end of file
+To get our agent to behave more like a human there are some enhancements we could make as well. Our current agent has a very specific strategy which will execute in exactly the same order most of the time. This is something that could be improved, as this makes the agent predictable, where a human would act more randomly. The agent has also very fast decision making, where a human would take its time to respond, the agent responds immediately to most things. When playing as a human this seems a bit odd, and a delay should be introduced.
```

Comment:
instead of randomly maybe use sporadic when describing the behaviour of a human player.
"The agent has also very fast decision making"should probably be "The agent also has very fast decision making"